src/Kliva/Views/StatsPage.xaml

```diff
@@ -14,7 +14,7 @@
     <Page.Resources>
         <DataTemplate x:Key="StatisticsDataTemplate">
             <UserControl>
-                <StackPanel Margin="8,0,8,12">
+                <StackPanel Margin="8,0,8,12" x:Name="RootStackPanel">
```

Comment:
Small guideline as well could you move all 'settings' inside the Visual State Setters. Instead of assuming this default one will fit the correct screen width.
So no margin definition on the ui element itself if it is tweaked in the setters...